### ISSUE TITLE
Creating a new single-project C# app won't run out of the box

### ISSUE THREAD

**Issue by dotMorten** [microsoft/WindowsAppSDK#1571]
**Describe the bug**
When you create new WinUI single-project packaged app using the 1.0.0-preview2 VSIX, and immediately hitting Run / F5, the app fails to run:
![image](https://user-images.githubusercontent.com/1378165/136627603-c7d349b4-e777-4400-b233-b4c96c1fee33.png)

The project isn't checked for deploy, and requires extra user interaction to configure the app.
![image](https://user-images.githubusercontent.com/1378165/136627470-fb29c2c9-cd28-46bb-aa8c-b21dccd7a6ba.png)
I'd expect a new project is able to immediately run from the beginning.

After checking this, however I'm met with a new problem:
![image](https://user-images.githubusercontent.com/1378165/136627671-acc14698-2381-4262-bd3f-76b5478fc3a6.png)

```
Creating a new clean layout...
Copying files: Total 116 mb to layout...
Checking whether required frameworks are installed...
Framework: Microsoft.WindowsAppRuntime.1.0-preview2/X86, app package version 0.272.1934.0 is not currently installed.
Framework: Microsoft.WindowsAppRuntime.1.0-preview2/X86, app package version 0.272.1934.0 is not currently installed.
Installing missing frameworks...
DEP0800: The required framework "C:\Users\mn\.nuget\packages\microsoft.windowsappsdk\1.0.0-preview2\build\..\tools\MSIX\win10-x86\Microsoft.WindowsAppRuntime.1.0-preview2.msix" failed to install. [0x80073CF9] Deployment Add operation with target volume C: on Package Microsoft.WindowsAppRuntime.1.0-preview2_0.272.1934.0_x86__8wekyb3d8bbwe from:  (Microsoft.WindowsAppRuntime.1.0-preview2.msix)  failed with error 0x800700C1. See http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app deployment issues.
1>Deployment of the application to the target device failed.
========== Deploy: 0 succeeded, 1 failed, 0 skipped ==========
```

Switch to x64 and app fails to deploy. Yet again I have to go and check on deploy for this configuration as well (and also for all the release configs).
Now magically the app will run.
Switch back to x86 and oddly this will run as well now too.

I don't think this is an acceptable first-run experience.

**Steps to reproduce the bug**
1. Install preview 2 VSIX for 2022 preview 4.1
2. Create new WinUI project "Blank App, Packaged (WinUI 3 in Desktop)"
3. Press F5 to attempt to run app.

**Expected behavior**
App runs on first try.

**Version Info**
NuGet package version: 
 [Microsoft.WindowsAppSDK 0.1.0-prerelease2] 
Visual Studio 2022 preview4.1

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |

**Additional context**

**Comment by seven-mile** [comment]
This seems to be a [known issue](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/preview-channel#other-limitations-and-known-issues) to be fixed in the next VS2022 Preview Release: You will succeed at the second deployment. Sounds like older preview version and VS2019 won't encounter this problem.

**Comment by dotMorten** [comment]
@seven-mile thanks but that only covers half of the issue here, and the issue in the release notes isn't afaik here (I'm guessing there's a large internal "secret" list of issues we can't see).

**Comment by andrewleader** [comment]
Thanks @dotMorten, this is captured in the release notes as a known issue as seven-mile mentioned. We're indeed tracking this as a ship-blocker for 1.0. We'll use this issue as the public issue to track this.

**Comment by andrewleader** [comment]
This is fixed in the upcoming 1.0 GA! Fixed by #1637
